setup.cfg

```diff
@@ -3,3 +3,9 @@ max-line-length: 99
 
 [flake8]
 max-line-length: 99
+
+[tool:pytest]
```

Comment:
`pytest`? But we use python unittest...

Reply:
yeah I wasn't able to setup `unittest`, I'm gonna give it another try. pytest seems to also work well when running the tests.

![image](https://user-images.githubusercontent.com/1882507/65951668-87c4af80-e40e-11e9-826c-ca3d472b1b37.png)